target/linux/ath79/image/generic.mk

```diff
@@ -171,6 +171,24 @@ define Device/comfast_cf-e120a-v3
 endef
 TARGET_DEVICES += comfast_cf-e120a-v3
 
+define Device/comfast_cf-wr650ac-v1
+  ATH_SOC := qca9558
+  DEVICE_VENDOR := COMFAST
+  DEVICE_MODEL := CF-WR650AC
+  DEVICE_VARIANT := v1
+  DEVICE_PACKAGES := kmod-ath9k kmod-ath10k-ct ath10k-firmware-qca988x-ct kmod-usb2 kmod-usb-ledtrig-usbport kmod-leds-reset kmod-mdio-gpio kmod-swconfig kmod-fs-squashfs kmod-leds-gpio kmod-ledtrig-gpio kmod-ledtrig-netdev kmod-rfkill om-watchdog
```

Comment:
Don't think you really have to pull all of that ...

Reply:
but yes indeed